packages/service-library/src/servicelib/_utils_profiling_middleware.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+
+def append_profile(body: str, profile: str) -> str:
+    try:
+        json.loads(body)
+        body += "\n" if not body.endswith("\n") else ""
+    except json.decoder.JSONDecodeError:
+        pass
+    body += json.dumps({"profile": profile})
+    return body
+
+
+def check_response_headers(
+    response_headers: dict[bytes, bytes]
+) -> list[tuple[bytes, bytes]]:
+    original_content_type: str = response_headers[b"content-type"].decode()
+    assert original_content_type in {
+        "application/x-ndjson",
+        "application/json",
+    }  # nosec
+    headers: dict = {}
+    headers[b"content-type"] = b"application/x-ndjson"
```

Comment:
- Let's extend and reuse these constants from [servicelib.mimetype_constants](https://github.com/itisfoundation/osparc-simcore/blob/fed9486caf5b2b19e1379c7f912e7a5b56da1507/packages/service-library/src/servicelib/mimetype_constants.py)

Reply:
very good point. will do